code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1432,6 +1560,31 @@ datum
 							for(var/j = 1, j <= rand(1, 3), j++)
 								step(B, pick(NORTH,SOUTH,EAST,WEST))
 
+		slimematerials
+			name = "Slime Materials"
+			id = "m_mats"
+			result = null
+			required_reagents = list("carbon" = 5)
+			result_amount = 1
+			required_container = /obj/item/slime_extract/silver
+			required_other = 1
+			on_reaction(var/datum/reagents/holder)
+				var/obj/item/stack/sheet/S1 = pick(new /obj/item/stack/sheet/plasteel,
```

Comment:
Due to the way it is coded BYOND might actually be dumb and create a new item for every possible item but only use 1, this code might be cleaner:

``` DM
var/obj/item/stack/sheet/S1 = new pick(/obj/item/stack/sheet/plasteel,
	/obj/item/stack/sheet/metal,
	/obj/item/stack/sheet/mineral/plasma,
	/obj/item/stack/sheet/mineral/silver,
	/obj/item/stack/sheet/mineral/gold,
	/obj/item/stack/sheet/mineral/uranium
)
```